inventory/src/main/java/io/axoniq/demo/hotel/inventory/query/RoomEntityRepository.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2020-2020. AxonIQ
```

Comment:
The copyright is just "a little" off ;-)

Reply:
Curious: what's the custom with copyright notices for js/ts/tsx files and such?

Reply:
I don't know. What is Axon's policy? I have never understood the use of license headers (and thus updating them every year), while there is a LICENSE file in place, distributed with the code.

Reply:
Updated all the copyrights